CONTRIBUTING.md

```diff
@@ -62,3 +117,11 @@ Please open an issue if you would like to discuss anything that could be improve
 Removal changes will not be applied until they have been pending for a minimum of 1 week (7 days). This grace window benefits projects that may be going through a temporary transition but are otherwise worthy of being on the list.
 
 Thanks everyone!
+
+## How decision are made
+
+The official group of maintainers have the final decision on what PRs are accepted. Discussions are made openly in issues. Where there appear to be conflicts, decisions are made by a majority vote where 2/3s of the maintainers are considered a quorum.
+
+Official maintainers must commit to responding to at least one PR per month.
```

Comment:
Duke (dukex)
Dmitri Shuralyov (dmitshur)
Dobrosław Żybort (matrixik)
Dean Karn  (joeybloggs)
Kirill Danshin (kirillDanshin)
Felipe Oliveira (felipeweb)
Jessica Temporal (jtemporal)
Ceriath (ceriath)

All responded/committed a year ago.

Recently active maintainers were Avelino, appleboy, cassiobotaro, panjf2000, and me (phanirithvij).
So I don't know about responding once every month being a *must*.

Reply:
we can create an automated process to update the list of maintainers
https://github.com/avelino/awesome-go/blob/main/MAINTAINERS

Reply:
I am simply raising the question about what the expectations are here. For some, there is an advantage to being listed as a maintainer. You get a certain amount of noteriety maintaining a prestigious list like this. However, just like needing to maintain a Go project to be in the list, a maintainer should be contributing on-going effor to show up on the maintainers list. Asking for a PR review once a month is not much of an ask.

Reply:
I see a difference between contributor and maintainer.

- contributor: someone who has contributed (or contributes) to the project — I am and always will be grateful for all the people who helped awesome-go get here
- maintainer: people who are currently helping to maintain the project

I proposed to update the list of maintainers automatically for this difference